### ISSUE TITLE
GUI Interface.

### ISSUE THREAD

**Issue by aniketpanjwani** [aniketpanjwani/chomper#14]
##### Steps to reproduce the problem:

There is no GUI interface. 

##### Any other comments? What have you tried so far?

I've never created a GUI. The main options for potential frameworks are [Toga](https://pybee.org/project/projects/libraries/toga/), [Kivy](https://kivy.org/#home), and [PyQt](https://riverbankcomputing.com/software/pyqt/intro).

If you've got any thoughts on how to choose between them, I would greatly appreciate it.

##### System information

Operating system: (e.g. Linux Mint 18)

**Comment by miheerdew** [comment]
I have only used wxPython once. I imagine that chomper would not need a fancy GUI, so maybe choose a lightweight framework (or something with high fun factor) ? You create also create a web ui. 

I look forward to suggestions others may have.

**Comment by aniketpanjwani** [comment]
Thanks Miheer - I missed that wxPython. I agree that Chomper doesn't need a fancy GUI. I'm probably going to decide by trying to write a very simple implementation and seeing which I like best. I'm leaning towards PyQt5, but I'm open to suggestions.

**Comment by aniketpanjwani** [comment]
I think it's also important to keep in mind the ease of packaging when thinking about choosing any particular GUI framework.

**Comment by miheerdew** [comment]
Agreed. There is also [tkinter](https://docs.python.org/3/library/tk.html) if you don't want external dependencies.
